RecoTracker/SpecialSeedGenerators/python/inOutSeedsFromTrackerMuons_cfi.py

```diff
@@ -21,3 +21,6 @@
     RefitRPCHits = cms.bool(True),
     Propagator = cms.string('SmartPropagatorAnyRKOpposite'),
 )
+
+from Configuration.ProcessModifiers.run4_PixelCPEGeneric_cff import run4_PixelCPEGeneric
+run4_PixelCPEGeneric.toModify(inOutSeedsFromTrackerMuons, TrackerRecHitBuilder = 'WithTrackAngle') # FIXME
```

Comment:
what is the purpose of "FIXME" here?

Reply:
copy-paste error, from undoing https://github.com/cms-sw/cmssw/pull/28448/commits/aeffc457daf6341d546449b19535f0f557a9d772. 
@duartej please clean this.

Reply:
done